cvat-core/src/server-proxy.ts

```diff
@@ -2167,6 +2200,75 @@ async function receiveWebhookEvents(type: WebhookSourceType): Promise<string[]>
     }
 }
 
+async function getQualityReports(filter): Promise<SerializedQualityReportData[]> {
+    const params = enableOrganization();
```

Comment:
``enableOrganization`` works automatically. 
It is not necessary to add it explicitly in simple requests.

It is necessary for example when creating a task, or exporting a dataset (where organization can be changed in theory during the process)